docs/src/app/(website)/page.tsx

```diff
@@ -0,0 +1,451 @@
+                      <PlusIcon className="AccordionIcon AccordionIconPlus" />
+                      <MinusIcon className="AccordionIcon AccordionIconMinus" />
+                    </Accordion.Trigger>
+                  </Accordion.Header>
+                  <Accordion.Panel className="AccordionPanel">
+                    <p className="Text size-2">
+                      Yes. Base UI is licensed under the MIT license, and is free for commercial
+                      use. You are free to use it in your commercial projects, and to modify it to
+                      suit your needs.
+                    </p>
+                  </Accordion.Panel>
+                </Accordion.Item>
+                <Accordion.Item className="AccordionItem">
+                  <Accordion.Header className="AccordionHeader">
+                    <Accordion.Trigger className="AccordionTrigger Text size-2">
+                      Do you offer enterprise SLAs?
+                      <PlusIcon className="AccordionIcon AccordionIconPlus" />
+                      <MinusIcon className="AccordionIcon AccordionIconMinus" />
+                    </Accordion.Trigger>
+                  </Accordion.Header>
+                  <Accordion.Panel className="AccordionPanel">
+                    <p className="Text size-2">
+                      Not currently. We do provide dedicated support channels to some very large
+                      enterprise companies who are working with us as design partners. But we do not
+                      currently provide Service Level Agreements, guaranteed response times, issue
+                      escalation, feature prioritization, or any other formal support guarantees.
+                    </p>
+                  </Accordion.Panel>
+                </Accordion.Item>
+              </Accordion.Root>
+            </div>
+          </section>
+        </main>
+        <div className="gcs-1 gce-9 bp3:gcs-3">
+          <div className="Separator" role="separator" aria-hidden="true"></div>
+        </div>
+        <footer className="d-c">
+          <div className="gcs-1 gce-9 bp2:gce-3">
+            <span className="Text size-1">© Base UI</span>
+          </div>
+          <nav className="d-f fd-c g-2 gcs-1 gce-9 bp2:gcs-3 bp4:gce-7" aria-label="social links">
+            <a className="Text size-1 Link" href="https://x.com/base_ui">
+              X
+            </a>
+            <a className="Text size-1 Link" href="https://github.com/mui/base-ui">
+              GitHub
+            </a>
+            <a className="Text size-1 Link" href="https://discord.com/invite/g6C3hUtuxz">
+              Discord
+            </a>
+            <a className="Text size-1 Link" href="https://www.npmjs.com/package/@base-ui/react">
+              npm
+            </a>
+            <a
+              className="Text size-1 Link"
+              href="https://bsky.app/profile/did:plc:nwr6peuxqzdzlbi72qr5kldc"
+            >
+              BlueSky
```

Comment:
```suggestion
              Bluesky
```

Do we even still use it?